core/domain/feedback_services.py

```diff
@@ -359,9 +547,15 @@ def get_feedback_message_references(user_id):
 
 
 def _add_feedback_message_reference(user_id, reference):
-    """Creates a new instance of UnsentFeedbackEmailModel or update existing
-    model instance for sending feedback message email."""
+    """Create a new instance of UnsentFeedbackEmailModel or update the
+    existing model for sending feedback message email.
 
+    Args:
+        user_id: str. If there's a UnsentFeedbackEmailModel for the given
+            user, update the instance with given reference, otherwise
+            create a new instance
+        reference: FeedbackMessageReference.
```

Comment:
Please describe this; the varname only explains its type but not its function/context. Something like "The new message reference to add to the buffer."